frontend/src/components/HomePage.js

```diff
@@ -40,19 +40,21 @@ export default class HomePage extends Component {
     }
 
     render() {
+        const fontSize = this.props.theme.typography.fontSize;
+        const fontSizeFactor = fontSize / 14; // default fontSize is 14
         return (
               <Router >
-                  <div className='appCenter'>
+                  <div className='appCenter' style={{height: window.innerHeight - 40*fontSizeFactor - 20, overflow:'auto'}}>
```

Comment:
Breaks the UI as the app is not vertically centered and the height is too tight (shows annoying scroll bar on the middle right side)